res/Sectors/M1105/Extolian.txt

```diff
@@ -31,447 +32,442 @@
+3102 (Filterers)          D948A9A-8 Hi In                        {  0 } (B93-4) [9A65] - - - 212 12 Na   M0 V                
+3103 Jedron               C645532-7 Ag Ni                        { -1 } (542-4) [6459] - M - 210 14 Na   G6 V                
+3105 Er Niskaphlaxac      D454464-7 Ni Pa Re O:3005              { -3 } (C32+1) [413B] - - - 514 15 HvFd F3 V                
+3106 Ka Cuqavackaap       C200405-C Ni Va                        {  0 } (A31+4) [245F] - - - 914 15 HvFd M0 V M3 V           
+3108 Andre                C777961-A Hi In Hive8 Huma2 Cy O:3209  {  3 } (989-2) [6C1B] - - - 203 11 HvFd K8 V                
+3109 Aokorurnanglir       C783203-9 Lo                           { -1 } (C11+3) [111A] - K - 802 11 HvFd K2 V                
+3203 Katana               A858778-A Ag                           {  3 } (46A-2) [5AAC] - K - 900 11 Na   K2 V                
+3204 (Sauna)              E884000-0 Ba                           { -3 } (A00-1) [0000] - - - 000 9  NaXX A7 V                
+3206 Loqaock              E777405-7 Ni Pa                        { -3 } (930-1) [6178] - - - 300 8  HvFd G5 V                
+3207 Fun                  D879301-7 Lo Huma3                     { -3 } (A21+0) [31A6] - - - 102 7  HvFd K2 V                
+3209 Sispo III            B444726-B Ag Pi Cp                     {  3 } (76B-1) [5A5F] - M - 414 13 HvFd G2 V                
+0114 Pisis                X7B6698-0 Fl Ni                        { -3 } (651+0) [3381] - - - 902 12 Cr   M5 V M2 V           
+0115 Iass Rsfii'kr        B463422-A Ni Cp                        {  1 } (933+4) [5568] - K - 500 4  Cr   G9 V K4 V           
+0116 Skia'mia             D967546-2 Ag Ni Pr                     { -2 } (B43-2) [7342] - - - 401 9  Cr   G3 V M5 V           
+0117 Carver               C300556-9 Ni Va                        { -1 } (642+4) [4427] - M - 600 6  NaXX M3 V                
+0214 Selmore              B552241-A Lo Po                        {  1 } (611+0) [2317] - - - 202 8  Na   G3 V                
+0312 Lead                 B405310-9 Lo Va Ic                     {  0 } (A21+0) [137C] - M - 221 16 NaXX K1 V                
+0318 Enfer                X9C4000-0 Fl Ba                        { -3 } (600+1) [0000] - - - 000 5  NaXX M4 V                
+0413 Hamius Farms         B642210-7 He Lo Po                     { -1 } (711-1) [1197] - K - 110 8  Na   F1 V A3 V           
+0420 Pakuucoon            A668402-B Ni Pa Cp                     {  1 } (634+1) [657F] - E - 100 7  HvFd A5 V M4 V           
+0519 Flandre              C437751-A                              {  1 } (669-3) [687A] - - - 512 12 Na   K0 V                
+0612 Dahaq                B2006S1-A Na Ni Va                     {  1 } (B56-2) [974F] - K - 904 10 HvFd K1 III K2 V         
+0613 Lunraot              E878725-5 Ag Pi                        { -1 } (A66+1) [B648] - - - 104 16 HvFd K1 V 
```

Comment:
3017 Esk Qoo  - This wants to be 'Cs' code (Subsector Capital)

Reply:
Doesn't Cp = subsector capital?